setup1-verifier/src/main.rs

```diff
@@ -16,57 +18,69 @@ fn development() -> Environment {
     .into()
 }
 
-#[inline]
 fn inner() -> Environment {
     Production::from(Parameters::AleoInner).into()
 }
 
-#[inline]
 fn outer() -> Environment {
     Production::from(Parameters::AleoOuter).into()
 }
 
-#[inline]
 fn universal() -> Environment {
     Production::from(Parameters::AleoUniversal).into()
 }
 
+#[derive(Debug, StructOpt)]
+#[structopt(name = "Aleo setup verifier")]
+struct Options {
+    #[structopt(long, help = "Path to a file containing verifier view key")]
+    view_key: PathBuf,
+    #[structopt(long, help = "Coordinator api url, for example http://localhost:9000")]
+    api_url: Url,
+}
+
+async fn request_coordinator_public_settings(coordinator_url: &Url) -> anyhow::Result<PublicSettings> {
+    let settings_endpoint_url = coordinator_url.join("/v1/contributor/settings")?;
```

Comment:
is it intentional that the url here is `/v1/contributor/settings`, `contributor` for the verifier implementation?

Reply:
It's a question of naming the settings endpoint. Initially I used `contributor/settings`, but now I think it's better to use `coordinator/settings`. So I would implement it like this, and then rename the endpoint in the coordinator. And the endpoint will be the same for both contributor and verifier.

Reply:
Fixed in #272